src/pages/saksbehandling/steg/beregningOgSimulering/beregning/Beregning.tsx

```diff
@@ -350,4 +355,28 @@ const Beregning = (props: VilkårsvurderingBaseProps) => {
     );
 };
 
+function erFradragLike(fradrag: Fradrag[] | undefined, formFradrag: FradragFormData[]): boolean {
+    if (!fradrag) return false;
+
+    const fradragSomFormData = zip(
+        FradragTilFradragFormData(fradrag.filter((f) => f.type !== Fradragstype.ForventetInntekt)),
+        formFradrag
+    );
+
+    if (fradragSomFormData.length != formFradrag.length) return false;
+
+    return fradragSomFormData.every(
+        (p) =>
+            p[0].beløp === p[1].beløp &&
+            p[0].fraUtland === p[1].fraUtland &&
+            p[0].periode?.fraOgMed?.toString() === p[1].periode?.fraOgMed?.toString() &&
+            p[0].periode?.tilOgMed?.toString() === p[1].periode?.tilOgMed?.toString() &&
+            p[0].tilhørerEPS === p[1].tilhørerEPS &&
+            p[0].type === p[1].type &&
+            p[0].utenlandskInntekt.beløpIUtenlandskValuta === p[1].utenlandskInntekt.beløpIUtenlandskValuta &&
+            p[0].utenlandskInntekt.kurs === p[1].utenlandskInntekt.kurs &&
+            p[0].utenlandskInntekt.valuta === p[1].utenlandskInntekt.valuta
+    );
+}
```

Comment:
Hvis du mekker en Eq-instans for fradrag så tror jeg hele denne funksjonen kunne ha blitt gjort enklere med https://gcanti.github.io/fp-ts/modules/Array.ts.html#geteq.